src/components/header/Header.js

```diff
@@ -1,19 +1,24 @@
 import styled from 'styled-components/macro'
+import logo from '../../assets/images/logo.svg'
 
 export default function Header() {
   return (
     <Wrapper>
-      <h1>Allaround Family</h1>
+      <Logo src={logo} alt="Allaround Family logo" />
```

Comment:
Please leave out the `logo` in the `alt` text. The user who is using a screenreader won't need to know that this is a `logo`. You could also think about having the `Logo` within an `h1`.
```suggestion
     <h1>
      <Logo src={logo} alt="Allaround Family" />
     </h1>
```